### DISCUSSION TITLE
Vitesse de lecture plus fine.

### DISCUSSION THREAD

**Discussion by RomainCahuNormandie** [EsupPortail/Esup-Pod#761] [Ideas]
Bonjour, 

Je suis prof d'education musicale en normandie et membre du Pôle de compétences disciplinaires de l'académie de Normandie.
Une des utilisations que nous faisons de Youtube, c'est la possibilité d'y poser des vidéos «guide-voix» pour que les élèves puissent travailler le chant hors de la classe : https://www.youtube.com/watch?v=4JJIFxBtmb8

Le lecteur youtube permet de pouvoir régler la vitesse de lecture de manière assez fine, à 5% près. Est-ce que c'est une possibilité envisageable pour Podeduc ?

**Comment by ptitloup** [comment]
Bonjour, 
je transfert votre demande à la DNE, qui a en charge votre Pod.
Nous verrons avec eux et reviendrons vers vous
Bien cordialement

**Comment by BHoury** [comment]
Bonjour,

Nous avons modifié le paramétrage.
Maintenant vous avez la possibilité de régler la vitesse avec ces valeurs : x0.25, x0.5, x0.75, x1, x1.25, x1.5, x1.75, x2
Cordialement
